docs/graphql/core/migrations/advanced/resetting-migrations.rst

```diff
@@ -0,0 +1,70 @@
+.. meta::
+   :description: Resetting Hasura migrations
+   :keywords: hasura, docs, migration, reset migrations, clear migrations
+
+.. _reset_migration:
+
+Reset Hasura migrations
+=======================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+Introduction
+------------
+
+If you have a lot of migrations and you need to squash them or your current migration state on the local machine is corrupted, you can reset the state and create new migrations from the state that is on the server.
```

Comment:
```suggestion
This page explains how to reset the state of migrations and create new migrations from the state that is on the server. This can be useful if the current migration state on your local machine is corrupted.
```

Reply:
Explanation: since this blog post came out, we have introduced a command to squash migration. Therefore, we shouldn't name it here as a scenario for this.